docs/on_chain_views.md

```diff
@@ -138,6 +148,37 @@ Tezos.contract.at(contractCallFib)
   })
   .catch((error) => println(`Error: ${JSON.stringify(error, null, 2)}`));
 ```
+</TabItem>
+  <TabItem value="walletAPI">
+
+```js live noInline
```

Comment:
In order to use the BeaconWallet, the live examples need a `wallet` props: `js live noInline wallet` instead of  `js live noInline`. Otherwise, the LegacyWallet is used which depends on a signer.

Reply:
i have fixed all of these and checked that they call the wallet when run.

the samples for "estimate" did not work ith the js live online wallet - this needs to be investigated

Reply:
put the estimate example back to how it was. It won't work with a wallet as it is calling a contract